### PR TITLE
[APP-3061] QuestionAnswer Component

### PR DESCRIPTION
**Pull request by JamesIves**
Adds a `QuestionAnswer` component which renders a list of frequently asked questions in a list.

* Look at the story in Storybook in multiple browsers.

Most comprised of expand collapse overrides to make it more suitable for this use case.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-3061)

### CODE REVIEW

File: components/molecules/QuestionAnswer/QuestionAnswer.scss [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          .uic--ec {
+            border-bottom: 0;
+            margin-bottom: 0;
+          }
+        }
+      }
+    }
+
+    // Applies a series of style overrides to re-purpose the ExpandCollapse component.
+    .uic--ec {
```
Comment (RohitRajendran) [inline components/molecules/QuestionAnswer/QuestionAnswer.scss]: Can we actually add all the overrides to the expandcollapse stylesheet like we do for the card overrides? It’s more likely to be seen if someone makes changes to the expandcollapse as well

**Comment by RohitRajendran** [comment]
Looks like too much bottom spacing 
image

**Comment by RohitRajendran** [comment]
The animation feels a little slow

File: components/molecules/QuestionAnswer/QuestionAnswer.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    {
+      'uic--question-answer': true,
+    },
+    className
+  );
+
+  return (
+    <div className={containerClasses} style={style}>
+      <div className="uic--row">
+        <div className="uic--question-answer__header uic--d-flex uic--justify-content-center uic--align-items-center uic--col-6">
```
Comment (RohitRajendran) [inline components/molecules/QuestionAnswer/QuestionAnswer.js]: Doesn’t have the right font color

Comment (RohitRajendran) [reply components/molecules/QuestionAnswer/QuestionAnswer.js]: An h2 tag seems appropriate here

File: components/molecules/QuestionAnswer/QuestionAnswer.story.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import QuestionAnswer from './QuestionAnswer';
+import QuestionAnswerReadMe from './QuestionAnswer.md';
+
+const stories = storiesOf('Molecules/QuestionAnswer', module);
+
+stories.addDecorator(withReadme(QuestionAnswerReadMe));
+
+const defaultProps = () => ({
+  header: 'FAQs',
+  items: [
```
Comment (RohitRajendran) [inline components/molecules/QuestionAnswer/QuestionAnswer.story.js]: Where the knobs at?

File: components/atoms/ExpandCollapse/ExpandCollapse.js [hunk 4]
```diff
@@ -24,10 +24,22 @@ class ExpandCollapse extends Component {
...
+  /** Handles the resizing event when the user makes the screen larger/smaller.
+   * @returns {undefined}
+   */
+  handleResize() {
+    return this.forceUpdate();
   }
 
   /** @inheritdoc */
   componentDidMount() {
+    if (isWindowDefined) {
```
Comment (CalMlynarczyk) [inline components/atoms/ExpandCollapse/ExpandCollapse.js]: `isWindowDefined` is a function.

File: components/molecules/QuestionAnswer/QuestionAnswer.scss [hunk 5]
```diff
@@ -0,0 +1,39 @@
+.uic--question-answer {
+  border-color: $light-gray;
+  border-style: solid;
+  border-width: 1px 0;
+  overflow: hidden;
+
+  .uic--question-answer__header {
```
Comment (CalMlynarczyk) [inline components/molecules/QuestionAnswer/QuestionAnswer.scss]: Weird that we're suddenly switching to BEM naming here.

File: components/atoms/ExpandCollapse/ExpandCollapse.js [hunk 6]
```diff
@@ -42,6 +54,13 @@ class ExpandCollapse extends Component {
     }
   }
 
+  /** @inheritdoc */
+  componentWillUnmount() {
+    if (isWindowDefined) {
```
Comment (CalMlynarczyk) [inline components/atoms/ExpandCollapse/ExpandCollapse.js]: Ditto

File: components/molecules/QuestionAnswer/QuestionAnswer.scss (hunk 5 above)
Comment (JamesIves) [reply components/molecules/QuestionAnswer/QuestionAnswer.scss]: There's a good amount of components that follow this pattern in the component library.

File: components/atoms/ExpandCollapse/ExpandCollapse.js (hunk 6 above)
Comment (JamesIves) [reply components/atoms/ExpandCollapse/ExpandCollapse.js]: Fixed!

**Pull request merged**